config/database.yml

```diff
@@ -6,59 +6,29 @@ default: &default
   username: postgres
   password: password
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
```

Comment:
Let's make sure not to include these changes in the PR since it seems to be specific to your system.